src/lib/utils/__tests__/formatDates-tests.tsx

```diff
@@ -0,0 +1,51 @@
+import { DateTime, Settings } from "luxon"
+import { formatDate, formatDateTime } from "../formatDates"
+
+const dateNow = 1525983752000 // Thursday, May 10, 2018 8:22:32.000 PM UTC in milliseconds
+let oneDayFromNow
+let oneYearFromNow
+
+beforeEach(() => {
+  jest.useFakeTimers()
+
+  // Thursday, May 10, 2018 8:22:32.000 PM UTC
+  Date.now = () => dateNow
```

Comment:
Same here, we can add a `const realNow = Data.now` at the top of the file and then `afterAll(() => Date.now = realNow)` in the test